allenact/utils/system.py

```diff
@@ -29,7 +29,7 @@ def get_logger() -> logging.Logger:
     logger: the `logging.Logger` object
     """
     if _new_logger():
-        _set_log_formatter()
+        init_logging("debug")
```

Comment:
I'm not sure if, by doing this, we make all children processes ignore the log level of the parent process. If you've tested it with e.g. `log_level=none` and it worked, I guess it's safe to keep the change.

Reply:
Good catch! I think I found a slightly better solution.

The problem that I was seeing was that if we didn't explicitly call `init_logging` before calling `get_logger` (e.g. this is often done if you have a script that uses an `allenact_plugin` without explicitly going through the `allenact.main` entry point) then any `print` statement would be ignored as the default log level was `warning`. I've now changed it so that if `init_logging` has not been called from the main process then it defaults to using a log level of "default". This seems to work for me now (log level is propagated to the subprocesses) but let me know if you foresee any other possible problems.